libtelemetry/src/test/java/com/mapbox/android/telemetry/AlarmSchedulerFlusherTest.java

```diff
@@ -67,4 +67,32 @@ public void checksAlarmUnregistered() throws Exception {
     verify(mockedAlarmManager, times(1)).cancel(eq(theAlarmSchedulerFlusher.obtainPendingIntent()));
     verify(mockedContext, times(1)).unregisterReceiver(mockedAlarmReceiver);
   }
+
+  @Test
+  public void checksAlarmUpdated() {
```

Comment:
looks like this test is verifying behavior of the `AlarmSchedulerFlusher` which is great, but what about `AlarmManager` itself? should we implement instrumentation test to make sure android framework is reacting properly to our change and there's no side effects of this change?